frontend/src/app/homepage.cljs

```diff
@@ -79,6 +93,26 @@
      (render-navigation-item "Upload" "#upload")
      (render-navigation-item "Container" "#container")]])
 
+(defn progress-width []
+  (min
+    (*
+      (float
+        (/
+          (:total_reports @backend-stats)
+    @report-target)) 100) 100))
+
+(defn render-stats []
+  [:div {:id "progressbar"}
+    [:div
+      [:h5 "Are we there yet?"]
+    ]
+    [:div {:id "progressbar-number"}
+      [:p (progress-width) "%"]]
+    [:div {
+      :id "progress"
+      :style {:width (str (progress-width) "%")}
+      }]])
```

Comment:
I'll need to set up some linter for the formatting. In comparison to e.g. python, AFAIK all lisps do:

- Ending brackets are not on their own lines. All of them are at the end of the last _command_
- We don't write opening bracket and then newline. e.g. `:div {\n`. The `{` goes to the same line as whatever comes after it

So for example:

```clojure
(defn render-stats []
  [:div {:id "progressbar"}
   [:h5 "Are we there yet?"]
   [:div {:id "progressbar-number"}
    [:p (progress-width) "%"]]
   [:div
    {:id "progress"
     :style {:width (str (progress-width) "%")}}]])
```

Especially the ending brackets are _incorrect_ on multiple places